drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -1456,11 +1456,10 @@ function DragAndDropBlock(runtime, element, configuration) {
             var container_height = $container.innerHeight();
 
             // We need to get the item position relative to the $container.
-            var item_offset = $item.offset();
             var container_offset = $container.offset();
             var original_position = {
-                left: drag_origin.x - container_offset.left,
-                top: item_offset.top - container_offset.top
+                left: drag_origin.x - container_offset.left - 50,
```

Comment:
why subtract `50`?

Reply:
On request from front-end team.
cc: @shahidrafique

Reply:
because we have 100px fixed size box.

Reply:
@nasirhjafri rather than using this magic number, can we add it to constant and use it here.